ledger/participant-integration-api/src/main/scala/platform/store/cache/MutableCacheBackedContractStore.scala

```diff
@@ -277,20 +266,15 @@ private[platform] class MutableCacheBackedContractStore(
         logger.debug(
           s"State events update: Archived(contractId=${contractId.coid}, globalKey=$globalKey, offset=$eventOffset, eventSequentialId=$eventSequentialId)"
         )
-      case LedgerEndMarker(eventOffset, eventSequentialId) =>
-        logger.debug(
-          s"Ledger end reached: $eventOffset -> $eventSequentialId"
```

Comment:
No substitute was added for this logging...do we really want to drop it?

Reply:
I think this acts as a substitute: https://github.com/digital-asset/daml/pull/14319/files#diff-f508d8081cc2e32c9e35300282b13f8973dae4e42ad9ac1fcc894175ced74f84R117